README.md

```diff
@@ -1,20 +1,21 @@
 <p align="center">
-	<img src="https://raw.githubusercontent.com/DeepRegNet/DeepReg/main/docs/asset/deepreg_logo_purple_beta.svg" alt="deepreg_logo" title="DeepReg" width="200" />
+	<img src="https://raw.githubusercontent.com/DeepRegNet/DeepReg/main/docs/asset/deepreg_logo_purple.svg" alt="deepreg_logo" title="DeepReg" width="200" />
 </p>
 
 # DeepReg
 
-[![Build Status](https://travis-ci.com/DeepRegNet/DeepReg.svg?branch=main)](https://travis-ci.com/DeepRegNet/DeepReg)
-[![Documentation Status](https://readthedocs.org/projects/deepreg/badge/?version=latest)](https://deepreg.readthedocs.io/en/latest/?badge=latest)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![Build Status](https://travis-ci.com/DeepRegNet/DeepReg.svg?branch=main)](https://travis-ci.com/DeepRegNet/DeepReg)
```

Comment:
What's the diff here? Is it just the order of badges? I have read over the urls a couple of time but can't spot the difference.

Reply:
Bumped Licence and Black badge to the front, put pypi near the JOSS badge.

Trying to group things (unchanging badges together, status badges together, "external link" badges together)